src/App.tsx

```diff
@@ -15,6 +16,7 @@ function  App(){
                 <Route path='/' component={LogIn} exact/>
                 <Route path='/wiki/:id' component={Wiki} />
                 <Route path='/home' component={HomePage} exact/>
+                <Route path='/signUp' component={SignUp}/>
```

Comment:
do we wanna use camel case for the routes? just curious because 'personalpage' doesn't, just something we can quickly decide in discord